packages/pigeon/lib/dart_generator.dart

```diff
@@ -103,11 +113,12 @@ void generateDart(Root root, StringSink sink) {
         indent.writeln('${field.dataType} ${field.name};');
       }
       indent.writeln('// ignore: unused_element');
-      indent.write('Map _toMap() ');
+      indent.write('Map<dynamic, dynamic> _toMap() ');
       indent.scoped('{', '}', () {
-        indent.writeln('Map dartleMap = Map();');
+        indent.writeln(
+            'final Map<dynamic, dynamic> dartleMap = <dynamic, dynamic>{};');
```

Comment:
This is going to turn into production code right? Can we traverse the code again and s/dartle/pigeon if needed?

Reply:
done